presto-hive/pom.xml

```diff
@@ -45,6 +45,12 @@
         <dependency>
             <groupId>com.facebook.hive</groupId>
             <artifactId>hive-dwrf</artifactId>
+            <exclusions>
+                <exclusion>
+                    <groupId>org.iq80.snappy</groupId>
+                    <artifactId>snappy</artifactId>
+                </exclusion>
```

Comment:
Why are we excluding this?

Reply:
There was an upper bound dependencies error for org.iq80.snappy:snappy:0.2 from this module(Attaching below for reference) : 

+-com.facebook.presto:presto-hive:0.291-SNAPSHOT
  +-com.facebook.hive:hive-dwrf:0.8.5
    +-org.iq80.snappy:snappy:0.2
and
+-com.facebook.presto:presto-hive:0.291-SNAPSHOT
  +-com.facebook.presto:presto-hive-metastore:0.291-SNAPSHOT
    +-org.iq80.snappy:snappy:0.5
and
+-com.facebook.presto:presto-hive:0.291-SNAPSHOT
  +-com.facebook.presto:presto-hive-metastore:0.291-SNAPSHOT
    +-org.iq80.snappy:snappy:0.5